<launch>
	<arg name="gui" default="False" />
	<param name="robot_description" textfile="$(find agv)/urdf/agv_disc_.urdf" />
	<param name="use_gui" value="$(arg gui)"/>
	<!-- JOY NODE -->
	<node respawn="true" pkg="joy"
		type="joy_node" name="joy_node" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
	</node>
	<!-- ROBOT STATE PUBLISHER NODE -->
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<!-- RVIZ -->
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find agv)/urdf.rviz" />
	<!-- PS3 JOYSTICK PARAMETERS -->
	<param name="axis_linear_L" value="1" type="int"/>
	<param name="axis_linear_R" value="3" type="int"/>
	<!-- SCALAR PARAMETERS -->
	<param name="scale_linear_L" value="9" type="double"/>
	<param name="scale_linear_R" value="9" type="double"/>
	<!-- LEFT&RIGHT WHEEL ANG VELOCITY PARAMETERS -->
	<param name="wLeft" value="0" type="double"/>
	<param name="wRight" value="0" type="double"/>
	<!-- JOINT STATE PUBLISHER NODE -->
	<node pkg="agv" type="jsPublisher"  name="Joint_State_Publisher" ></node>
	<!-- TF BROADCASTER NODE -->
	<node pkg="agv" type="odomTF_better"  name="Odometry_Tf_Broadcaster" ></node>
</launch>

